vetiver/server.py

```diff
@@ -274,14 +260,14 @@ def predict(endpoint, data: Union[dict, pd.DataFrame, pd.Series], **kw):
     return response_df
 
 
-def _prepare_data(pred_data):
+def _prepare_data(pred_data: list) -> list:
```

Comment:
The typing can be more specific. For a start, this can do

```python
def _prepare_data(pred_data: list[dict[str, Any]]) -> list[Any]:
```

though eventually the `Any` in this case could be replaced with a defined type for the input value type.